source/discovery/carla-discovery.cpp

```diff
@@ -1676,7 +1676,7 @@ int main(int argc, char* argv[])
 #ifdef USE_JUCE_PROCESSORS
         do_juce_check(filename, "VST3", doInit);
 #else
-        DISCOVERY_OUT("error", "VST3 support not available");
```

Comment:
this does not work, needs to be reverted. my native vst dicovery is only for vst2